.test/.external-pins/library/alpine___3.16

```diff
@@ -0,0 +1 @@
+sha256:e4cdb7d47b06ba0a062ad2a97a7d154967c8f83934594d9f2bd3efa89292996b
```

Comment:
Aren't these versions out of support?

Reply:
Alpine 3.16 is still supported (it's 3.15 that just went EOL), but pinning like this ensures our tests are reproducible (we'll be treating `alpine:3.16` as `sha256:e4cdb7d47b06ba0a062ad2a97a7d154967c8f83934594d9f2bd3efa89292996b` in the future for the purposes of these tests even if it's updated).